converters/toZigbee.js

```diff
@@ -5004,15 +5006,18 @@ const converters = {
     },
     wiser_sed_thermostat_local_temperature_calibration: {
         key: ['local_temperature_calibration'],
-        convertSet: async (entity, key, value, meta) => {
-            globalStore.putValue(entity, 'localTemperatureCalibrationUpdated', true);
+        convertSet: (entity, key, value, meta) => {
+            entity.write('hvacThermostat', {localTemperatureCalibration: Math.round(value * 10)},
+                {srcEndpoint: 11, disableDefaultResponse: true, sendWhenActive: true});
             return {state: {local_temperature_calibration: value}};
         },
     },
     wiser_sed_thermostat_keypad_lockout: {
         key: ['keypad_lockout'],
         convertSet: async (entity, key, value, meta) => {
-            globalStore.putValue(entity, 'keypadLockoutUpdated', true);
+            const keypadLockout = utils.getKey(constants.keypadLockoutMode, value, value, Number);
+            await entity.write('hvacUserInterfaceCfg', {keypadLockout},
```

Comment:
using `sendWhenActive` with entity.write appears to defer the dispatch, however when the device wakes up it gets stuck in a loop and is constantly sending the write attribute command over the air.

Reply:
This looping issue seems to be resolved with the latest fix in zigbee-herdsman 👍